golem_messages/validators.py

```diff
@@ -19,3 +19,133 @@ def validate_integer(field_name, value):
             field=field_name,
             value=value,
         )
+
+
+def is_number_positive(value, value_type):
+    return isinstance(value, value_type) and value > 0
```

Comment:
If this will be called like ie. `is_number_positive('aaa', str)` it will raise exception. If that is ok, then ok, otherwise maybe validate if `value_type` is one of expected types?

Reply:
Added assert to be sure that function can handle only integers and floats